util/dvsim/SimCfg.py

```diff
@@ -463,6 +464,13 @@ def gen_results_sub(items, results, fail_msgs):
                 regr_results=regr_results,
                 map_full_testplan=self.map_full_testplan)
             results_str += "\n"
+            self.final_total = self.testplan.final_total
+            # if testplan name and dut name are different, show both
+            if self.final_total[0].upper() != self.dut.upper():
+                self.final_total[0] += '/' + self.dut.upper()
+            # pass_rate
+            self.final_total[-1] = self.append_result_link(
```

Comment:
add one more column, pleas check this. It may look nicer if we have both color and link on pass_rate.
https://reports.opentitan.org/hw/top_earlgrey/dv/summarys.html